config-default.yml

```diff
@@ -138,7 +139,7 @@ guild:
 
     staff_channels: [*ADMINS, *ADMIN_SPAM, *MOD_SPAM, *MODS, *HELPERS, *ORGANISATION, *DEFCON]
     ignored: [*ADMINS, *MESSAGE_LOG, *MODLOG]
-    attachment_repost: 649243850006855680
+    attachment_repost: *ATTCH_LOG
```

Comment:
This is redundant. It should be removed and replaced by `attachment_log` entirely in the code.